eclipse/src/saros/filesystem/EclipseWorkspaceRootImpl.java

```diff
@@ -1,26 +1,11 @@
-/** */
 package saros.filesystem;
 
-import java.util.ArrayList;
-import java.util.List;
+import org.eclipse.core.resources.IWorkspaceRoot;
 
-/** Eclipse implementation of {@link IWorkspaceRoot}. */
-public class EclipseWorkspaceRootImpl extends EclipseContainerImpl implements IWorkspaceRoot {
+/** Eclipse {@link IContainer} implementation representing an {@link IWorkspaceRoot}. */
```

Comment:
Is this class deleted in another patch ?

Reply:
No, I didn't remove it because it is still used in the eclipse implementation. If a file in the workspace root asks for its parent resource, a workspace root object is returned instead of a folder. This class is now used as a simple implementation of the abstract `EclipseContainerImpl`. But I could also remove the `abstract` modifier from `EclipseContainerImpl` and change the delegate type to `IContainer` instead of `IReosurce` if you would prefer.

Reply:
>  But I could also remove the abstract modifier from EclipseContainerImpl and change the delegate type to IContainer instead of IReosurce if you would prefer

Do you think the additional abstraction layer `EclipseWorkspaceRootImpl` provides valuable information? Otherwise, I would prefer your idea of removing the class and using the `EclipseContainerImpl`.

Reply:
> Do you think the additional abstraction layer `EclipseWorkspaceRootImpl` provides valuable information?

No, not really as I couldn't be bothered to added any new javadoc to it. Will adjust the container implementation and remove the class.

Reply:
Done.